modin/pandas/test/test_groupby.py

```diff
@@ -1111,6 +1119,55 @@ def eval___getattr__(modin_groupby, pandas_groupby, item):
     )
 
 
+def eval__getitem__(md_grp, pd_grp, item):
+    eval_general(
+        md_grp,
+        pd_grp,
+        lambda grp: grp[item].mean(),
+        comparator=build_types_asserter(df_equals),
+    )
+    eval_general(
+        md_grp,
+        pd_grp,
+        lambda grp: grp[item].count(),
+        comparator=build_types_asserter(df_equals),
+    )
+
+    def build_list_agg(fns):
+        def test(grp):
+            res = grp[item].agg(fns)
+            if res.ndim == 2:
+                # Modin's frame has an extra level in the result. Alligning columns to compare.
```

Comment:
This feels like a bug that we work around in the tests. Doesn't feel good.

Reply:
added a reference to an issue related with this bug